cli/src/merge_tools/diff_working_copies.rs

```diff
@@ -285,11 +286,7 @@ diff editing in mind and be a little inaccurate.
         let mut output_tree_state = diff_wc.output.unwrap_or(diff_wc.right);
         output_tree_state.snapshot(&SnapshotOptions {
             base_ignores,
-            fsmonitor_settings: FsmonitorSettings::None,
-            progress: None,
-            start_tracking_matcher: &EverythingMatcher,
-            max_new_file_size: u64::MAX,
-            conflict_marker_style,
+            ..SnapshotOptions::empty_for_test()
```

Comment:
I am not sure if we should use a test method outside tests. Why not just explicitly list all the fields just like the old code?

Reply:
I agree with @06393993. As the name suggests, `empty_for_test()` is intended for tests.